velox/docs/functions/spark/datetime.rst

```diff
@@ -7,9 +7,17 @@ Convenience Extraction Functions
 
 These functions support TIMESTAMP and DATE input types.
 
-.. spark:function:: year(x) -> integer
+.. spark:function:: make_date(int, int, int) -> date
```

Comment:
nit: make_date(year, month, day) for readability, then clarify that all inputs must be of type INTEGER.

Here is the documentation from Spark. Which version did you implement here? ANSI or not? Please, clarify here and in PR description.

Also, is month zero-based or 1-based. Is January 0 or 1?

```
make_date(year, month, day)

Create date from year, month and day fields. If the configuration `spark.sql.ansi.enabled` is false, the function returns NULL on invalid inputs. Otherwise, it will throw an error instead.
```